infra/main.bicep

```diff
@@ -0,0 +1,152 @@
+var rg = resourceGroup()
+var location = resourceGroup().location
+
+@description('Suffix to create uniqute resource names. 4-6 symbols. Default is random 6 symbols.')
+@minLength(4)
+@maxLength(6)
+param suffix string = substring(newGuid(), 0, 6)
+
+// //az deployment sub create -f main.bicep --location=$Location --parameters location=$Location -c
+// targetScope = 'subscription'
+// param location string
+// resource rg 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+//   name: 'km-rg-${suffix}'
+//   location: location
+// }
+
+module managedidentity 'teamplates/identity.bicep' = {
+  name: 'managedidentity-${suffix}'
+  scope: rg
+  params: {
+    location: location
+    suffix: suffix
+  }
+}
+
+module storage 'teamplates/storage.bicep' = {
+  name: 'storage-${suffix}'
+  scope: rg
+  params: {
+    location: location
+    suffix: suffix
+    managedIdentityPrincipalId: managedidentity.outputs.managedIdentityPrincipalId
+  }
+}
+
+module search 'teamplates/search.bicep' = {
+  name: 'search-${suffix}'
+  scope: rg
+  params: {
+    location: location
+    name: 'km-search-${suffix}'
+    suffix: suffix
+    managedIdentityPrincipalId: managedidentity.outputs.managedIdentityPrincipalId
+  }
+}
+
+@description('ChatGpt Deployment Capacity in thousands. Default is 1.')
+@minValue(1)
+@maxValue(40)
+param chatGptDeploymentCapacity int = 30
+
+var chatGpt = {
+  modelName: 'gpt-35-turbo-16k'
+  deploymentName: 'chat'
+  deploymentVersion: '0613'
+  deploymentCapacity: chatGptDeploymentCapacity
+}
+
+@description('Embedding Deployment Capacity in thousands. Default is 1.')
```

Comment:
I think this needs documentation to explain which capacity this is, and how to find or set the right value